### PR TITLE
Fixed a profile form verification bug

### PR DESCRIPTION
**Pull request by apluscs**
The profile form wasn't being verified before sending to backend. Now it is.

### CODE REVIEW

File: src/main/webapp/profile.js [hunk 1]
```diff
@@ -40,6 +42,12 @@ function submitProfileForm(methodType, redirectUrl) {
     });
 }
 
+function validateProfileForm() {
```
Comment (meagancheese) [inline src/main/webapp/profile.js:45]: Where is this called?

**Review by meagancheese — CHANGES_REQUESTED** [review]
One question about the validation process.

File: src/main/webapp/profile.js (hunk 1 above)
Comment (apluscs) [reply src/main/webapp/profile.js:45]: image
When they submit on profile or registration page

**Review by meagancheese — APPROVED** [review]
LGTM

**Review by GrantComm — APPROVED** [review]
LGTM

File: src/main/webapp/profile.html [hunk 2]
```diff
@@ -40,7 +40,8 @@
       </nav>
       <br>
       <h1 class="text-center">Your Profile</h1>
-      <form class="needs-validation" novalidate action="/person" method="POST">
+      <form class="needs-validation" id="profile-form" novalidate 
+          onsubmit="event.preventDefault(); submitProfileForm('PUT','/profile.html')">
```
Comment (jalexanderqed) [inline src/main/webapp/profile.html]: Semicolon after `submitProfileForm()`.

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**